recipes/libbacktrace/all/conanfile.py

```diff
@@ -0,0 +1,54 @@
+from conans import AutoToolsBuildEnvironment, ConanFile
+
+
+class LibbacktraceConan(ConanFile):
+    name = "libbacktrace"
+    description = "A C library that may be linked into a C/C++ program to produce symbolic backtraces."
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ianlancetaylor/libbacktrace"
+    license = "BSD-3-Clause"
+    topics = ("conan", "backtrace", "stack-trace")
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    __autotools = None
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    @property
+    def scm(self):
+        source = {"type": "git"}
+        source.update(**self.conan_data["sources"][self.version])
+        return source
```

Comment:
I do not think this method is supported by CCI 🤔

Reply:
libbacktrace doesn't provide releases.

Reply:
See https://github.com/ianlancetaylor/libbacktrace/issues/67

Reply:
It seems conandata.yml doesn't allow `revision` entry. Can this be relaxed? If allowed, we could use it to map the commit hashes to versions to enable reproducible builds. For example:
* 1.0-0: revision: xxxx
* 1.0-1: revision: yyyy

Reply:
There is a version schema for libraries that doesn't provide explicit versions: https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases

OTH, I think this is breaking the current CI process. When I run it locally I get an error

```
ERROR: libbacktrace/1.0: Cannot load recipe.
'property' object has no attribute 'update' 
```

Indeed, every _public_ function or member in the `ConanFile` is reserved to Conan, and `scm` is an attribute we are already using.

Reply:
Thanks!
https://github.com/conan-io/conan-center-index/pull/4431/commits/d24ad500c1d844f27ae38e5fd4d5706c30b090ac